src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -7366,10 +7366,15 @@ def create_new_cluster_with_monitoring_aad_auth(self, resource_group, resource_g
              self.cmd(get_cmd, checks=[
                 self.check('properties.dataSources.extensions[0].extensionSettings.dataCollectionSettings.namespaces[0]', f'kube-system')
             ])
-
+             self.cmd(get_cmd, checks=[
+                self.check('properties.dataSources.extensions[0].extensionSettings.dataCollectionSettings.streams[0]', f'Microsoft-ContainerLogV2')
+            ])
+             self.cmd(get_cmd, checks=[
+                self.check('properties.dataFlows[0].streams[0]', f'Microsoft-ContainerLogV2')
+            ])
```

Comment:
It is suggested to combine these checks in one command implementation, e.g., 

```python
self.cmd(get_cmd, checks=[
                self.check('properties.dataSources.extensions[0].extensionSettings.dataCollectionSettings.namespaces[0]', f'kube-system'),
                self.check('properties.dataSources.extensions[0].extensionSettings.dataCollectionSettings.streams[0]', f'Microsoft-ContainerLogV2'),
                self.check('properties.dataFlows[0].streams[0]', f'Microsoft-ContainerLogV2'),
            ])
```

Reply:
fixed.